index.html

```diff
@@ -0,0 +1,247 @@
+            <li class="item"><a href="#">Contact</a></li>
+            <li class="mobile-menu"><a href="#">
+                    <img src="assets/images/header-menu.svg" alt="Hamburguer Icon">
+                </a></li>
+        </ul>
+    </nav>
+    <section class="heading flex">
+        <h1 class="heading__title fw-700">I'm Adeyemi.<br /> Glad to see you!</h1>
+        <p class="heading__desc">
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <p class="heading__connect">LET’S CONNECT</p>
+        <div class="heading__icons">
+            <ul class="heading__icons__ul ul flex">
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/twitter.svg" alt="twitter icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/medium.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/github.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+            </ul>
+        </div>
+    </section>
+
+    <section class="tonic">
+        <img class="image is_desktop" src="assets/images/banners/desktop_tonic.svg" alt="tonic" />
+        <img class="image is_mobile" src="assets/images/banners/desktop_facebook.svg" alt="tonic" />
+
+        <div class="content">
+            <h1 class="title">Tonic</h1>
+
+            <p class="summary"><span>CANOPY</span><span class="dotted"></span><span class="color">Back End
+                    Dev</span><span class="dotted"></span><span class="color">2015</span></p>
+
+            <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+            <ul class="tagging ul flex">
+                <li>
+                    <button class="tag_button">html</button>
+                </li>
+                <li>
+                    <button class="tag_button">css</button>
+                </li>
+                <li>
+                    <button class="tag_button">javascript</button>
+                </li>
+            </ul>
+
+            <a href="#" class="button-outline">See Project</a>
+
+        </div>
+    </section>
```

Comment:
- [ ] The same thing here too. At breakpoint `768px` which is the starting point for the desktop version of your project does not display this section as expected. The image and the  image information take equal width in the card and this must be responsive across the device sizes.  Kindly check the screenshot of both designs and walk your design alongside the Figma design.

### Your Design (breakpoint: 1024px)
![your-project-card-desktop](https://user-images.githubusercontent.com/26597432/236409337-2e5f765b-0f54-4d8f-9f7d-be29c06c48bd.png)

### Your Design (breakpoint: 768px)

![your-project-card-768px](https://user-images.githubusercontent.com/26597432/236410190-1e46577f-f143-4904-bdac-dce3d9abe347.png)

### Figma Design
![temp-project-card-desktop](https://user-images.githubusercontent.com/26597432/236409385-9d64ef27-44d8-4f29-8b46-b8e5695c3c5e.png)